examples/petsc/README.md

```diff
@@ -32,9 +32,9 @@ In addition to the common arguments, the following arguments may be set:
 This code solves the CEED bakeoff problems on a unstructured grid using DMPlex
 with p-multigrid implemented in PCMG. This example requires a PETSc version later than 3.11.3.
 
-To build, run `make multigrid`.
+To build, run `make multigrid`
 
-To run, `./multigrid -ceed [ceed-resource] -problem bp[1-6] -degree [degree]`.
+To run, `./multigrid -ceed [ceed-resource] -problem bp[1-6] -degree [degree]`
 
 In addition to the common arguments, the following arguments may be set:
 
```

Comment:
@jeremy I can't tag line 46 here. But I'm wondering if "The following arguments can be specified for both examples:" refers to the bpsdmplex and multigrid examples, or actually to all of above. I can update it, if necessary.

Reply:
Those six arguments apply to bpsraw, bps, and multigrid.

Reply:
Ok then it was outdated. I can update it.

Ps: I'm not sure why Fortran tests would fail with Clang, since I've only updated this README in this commit

Reply:
PPs: I've removed the periods at the end of those lines in case an inexperienced user copies and pastes the Readme line to run, and does not realize the period is not necessary to run